app/build.gradle

```diff
@@ -9,11 +9,11 @@ plugins {
 }
 
 android {
-    compileSdk 32
+    compileSdk 33
 
     defaultConfig {
         minSdk 21
```

Comment:
Not for now because, Android 14 is not available.
And 3.0 version have a lot of bugs, i want push new release to fix bugs before remove support to Android 5 and 5.1